packages/msw-addon/src/mswDecorator.ts

```diff
@@ -1,16 +1,21 @@
 import { isNodeProcess } from 'is-node-process'
+import type {
+  StoryFn,
+  DecoratorFunction,
+  StoryContext,
+} from '@storybook/addons'
+
+import type { SetupWorkerApi, RequestHandler } from 'msw'
+import type { SetupServerApi } from 'msw/node'
 
 const IS_BROWSER = !isNodeProcess()
-let api
+let api: SetupWorkerApi | SetupServerApi
 
-export function initializeWorker(options) {
-  console.warn(
-    `[MSW] "initializeWorker" is now deprecated, please use "initialize" instead. This method will be removed in future releases.`
-  )
-  return initialize(options)
-}
+type InitializeOptions =
```

Comment:
Let's replace it once https://github.com/mswjs/msw/pull/994 is merged. I'm totally for exporting and reusing common type declarations.

Reply:
Awesome <3